packages/grid_client/scripts/kubernetes_mycelium.ts

```diff
@@ -0,0 +1,111 @@
+import { FilterOptions, K8SModel } from "../src";
+import { config, getClient } from "./client_loader";
+import { log } from "./utils";
+
+async function deploy(client, k8s) {
+  const res = await client.k8s.deploy(k8s);
+  log("================= Deploying K8s =================");
+  log(res);
+  log("================= Deploying K8s =================");
+}
+
+async function getDeployment(client, k8s) {
+  const res = await client.k8s.getObj(k8s);
+  log("================= Getting deployment information =================");
+  log(res);
+  log("================= Getting deployment information =================");
+}
+
+async function cancel(client, k8s) {
+  const res = await client.k8s.delete(k8s);
+  log("================= Canceling the deployment =================");
+  log(res);
+  log("================= Canceling the deployment =================");
+}
+
+async function main() {
+  const grid3 = await getClient();
+
+  const k: K8SModel = {
+    name: "testk8sMy",
+    secret: "secret",
+    network: {
+      name: "monNetwork",
+      ip_range: "10.238.0.0/16",
+      addAccess: true,
+      myceliumSeeds: [
+        {
+          nodeId: 153,
+          seed: "a5f0ea16a744af2c0c23fc878d727a6f355079f82d979ad4bc75dd8fb5ebc90e",
+        },
+        {
+          nodeId: 31,
+          seed: "7edd9c250f834cb326c3cf116040cf2214f38c669bf27a72e2f5b9e44fc7b27e",
+        },
+        {
+          nodeId: 177,
+          seed: "3162379d1d2530e2113ae2c7142370cc195628238fa1059f2f25ef51c113e892",
+        },
```

Comment:
let's make these seeds commented to make them feel they are optional but the user still knows how to add them.